# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/Cargo.toml
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 gfx/wr/webrender/Cargo.toml
--- a/gfx/wr/webrender/Cargo.toml	Fri Oct 03 18:01:30 2025 +0000
+++ b/gfx/wr/webrender/Cargo.toml	Fri Oct 03 18:08:29 2025 +0000
@@ -41,7 +41,7 @@
 plane-split = "0.18"
 png = { optional = true, version = "0.16" }
 rayon = "1"
-ron = { optional = true, version = "0.10.0" }
+ron = { optional = true, version = "0.11.0" }
 serde = { optional = true, version = "1.0", features = ["serde_derive"] }
 smallvec = "1"
 api = { version = "0.62.0", path = "../webrender_api", package = "webrender_api" }